distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -183,6 +183,12 @@ org.codice.ddf.platform.util.http.initialRetryInterval=10
 javax.xml.parsers.DocumentBuilderFactory=org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
 javax.xml.transform.TransformerFactory=org.apache.xalan.processor.TransformerFactoryImpl
 
+#
+# Metadata Size Limit
+#
+
+# Sets the size limit of the metadata within the metacards. Size is in bytes. Default is 5242880.
```

Comment:
✏️ I would replace `Default is 5242880` with `Default is 5MB` for clarity

Reply:
fixed